src/shared/constants/timeRanges.ts

```diff
@@ -135,3 +135,6 @@ export const SELECTABLE_USAGE_TIME_RANGES: SelectableDurationTimeRange[] = [
 
 export const DEFAULT_USAGE_TIME_RANGE = pastDayTimeRange
 export const DEFAULT_TIME_RANGE = pastHourTimeRange
+
+export const DESIRED_POINTS_PER_GRAPH = 360
+export const FALLBACK_WINDOW_PERIOD = 15000
```

Comment:
I thought the default for windowPeriod was 10s, not 15s?

Reply:
To be clear: this is the fallback if it doesn't recognize the timeRange used. Or something else fails. Most of the `SELECTABLE_TIME_RANGES` uses 10s as the windowPeriod.